app/views/collection/block_users.html.slim

```diff
@@ -0,0 +1,44 @@
+.litebox-embed(style="width:450px")
+  
+  h3 =t('collection.edit.block_users')
+  p.fglight =t('collection.edit.block_users_description')
```

Comment:
We should use a different message here, which is more verbose.  Something like:

_These users have been blocked from access to the collection. The collection will not be visible to them, as if it had been marked private or deleted.  Adding users to this list will not notify them that they have been blocked._